lib/lint.ml

```diff
@@ -102,11 +102,24 @@ let lint_opam_doc pkg =
   0
 
 let lint_opam_home_and_dev pkg =
-  lint_res
-    ~msgf:(fun l ->
-      l "opam fields %a and %a can be parsed by dune-release" pp_field
-        "homepage" pp_field "dev-repo")
-    (Pkg.distrib_user_and_repo pkg)
+  ( match Pkg.distrib_uri pkg with
+  | Error _ ->
+      report_status `Fail (fun l ->
```

Comment:
Since we're not going to exit 0 we should make it a warning I guess, what do you think?

We really need to think about the lint part and decide what should make something fatal or not. I'm thinking that maybe we should remove the lint step from the `distrib` command and run separately or as part of bistro. That way the `--skip-lint` would only be a flag for bistro and people unhappy with dune-release's linting can skip it if they know what they're doing.